BeerProject/src/main/java/wawer/kamil/beerproject/generators/Generator.java

```diff
@@ -1,22 +1,42 @@
 package wawer.kamil.beerproject.generators;
 
 import lombok.RequiredArgsConstructor;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
 import org.springframework.stereotype.Component;
 import wawer.kamil.beerproject.model.Address;
 import wawer.kamil.beerproject.model.Beer;
 import wawer.kamil.beerproject.model.Brewery;
+import wawer.kamil.beerproject.model.User;
 import wawer.kamil.beerproject.model.enums.StyleBeer;
 import wawer.kamil.beerproject.repositories.BreweryRepository;
 
+import java.time.LocalDateTime;
+
+import static wawer.kamil.beerproject.configuration.security.ApplicationUserRole.USER;
+
 @Component
 @RequiredArgsConstructor
 public class Generator {
 
     private final BreweryRepository repository;
 
-    //@EventListener(ApplicationReadyEvent.class)
-    public void addData(){
+    public static User createUser() {
+        PasswordEncoder passwordEncoder = new BCryptPasswordEncoder();
```

Comment:
PasswordEncoder bycha powinien być przez @Bean. Dlaczego tak jest jak powyżęj?

Reply:
już nie pamiętam dokładnie ale musiałem wywalić wstrzykiwanie przez konstruktor, ponieważ tworzyła mi się z tego zależność kołowa, a ponieważ ta metoda jest do celów developerskich i w szybki sposób stworzenia jakiegokolwiek usera to roboczo tak to rozwiązałem.

Reply:
ok rozumiem